packages/core/src/percy.js

```diff
@@ -404,6 +415,22 @@ export class Percy {
     if (syncMode) options.sync = syncMode;
     return syncMode;
   }
+
+  sendBuildEvents() {
+    // Only send cli error when PERCY_CLIENT_ERROR_LOGS is set to true
+    if (process.env.PERCY_CLIENT_ERROR_LOGS !== 'false') {
+      // percy token is present
+      if (process.env.PERCY_TOKEN) {
+        const logs = [...this.log.messageDetail()].map((item) => {return JSON.stringify(item)}).join('\n')
+        const content = base64encode(Pako.gzip(logs))
+        const eventObject = {
+          content: content
+        };
+        // console.log(content)
+        this.client.sendBuildEvents(this.build?.id, eventObject);
```

Comment:
this is an async function, you are not awaiting it, mostly events will never be sent in that case